src/ui/views/SendToken/index.tsx

```diff
@@ -303,11 +308,7 @@ const SendToken = () => {
   };
 
   const handleClickBack = () => {
-    if (history.length > 1) {
-      history.goBack();
-    } else {
-      history.replace('/');
-    }
+    history.replace('/');
   };
 
```

Comment:
When back from chain management, which replace chain management stack with send token, then there will be two send-back history on stack, therefore just replace to ("/")